README.md

```diff
@@ -114,7 +133,7 @@ Registers a callback to be called when an event of a type is emitted. Note that
 #### `events.emit`
 
 ```js
-events.emit(type, payload, ...args)
+events.emit(type, payload)
```

Comment:
seems like the `ctx` should go as an optional arg here? line 141 also needs to be updated

Reply:
I actually don't allow emitting events with ctx as a third argument. It is added by using `EventEmitter.of(ctx)`